lib/user.rb

```diff
@@ -0,0 +1,37 @@
+class User
+
```

Comment:
Maybe you could create an initialize method to store the name, username as instance variables so you could remove lines 11-21?